evalml/tests/pipeline_tests/test_pipeline_utils.py

```diff
@@ -197,10 +196,22 @@ def test_make_pipeline(
         ProblemTypes.TIME_SERIES_REGRESSION,
     ],
 )
-@pytest.mark.parametrize("frequency", ["D", "MS", "A", "T"])
+@pytest.mark.parametrize(
+    "frequency, should_decomp",
+    [
+        ("D", True),
+        ("MS", True),
+        ("A", False),
+        ("T", False),
+        ("10T", False),
+        ("AS-JAN", False),
+        ("YS", False),
+    ],
+)
```

Comment:
I suspect this is going to be on a per-decomposer basis...so we might need to do this differently going forward, when we introduce new decomposers and also introduce new ways of trying more pipelines with different decomposers.  I think it's fine for now and we can kick the can to when we do X11 or X13.

Could we also add mention that this is a test for time series only in the test name?